src/test/java/de/retest/recheck/RecheckImplIT.java

```diff
@@ -30,7 +31,11 @@
 	static final String MESSAGE_START_PATTERN = "A detailed report will be created at '.*'";
 	static final String MESSAGE_START_REPLACE = "A detailed report will be created at '*'";
 
-	static final Filter METADATA_FILTER = Filters.parse( "import: metadata.filter" );
+	static final Filter METADATA_FILTER = Filters.parse( Stream.of( // 
+			"import: metadata.filter", //
+			"attribute=os.name", // 
+			"attribute=os.version" //
+	) );
```

Comment:
That was clarified with #756 where the metadata.filter now includes **all** metadata instead of just the volatile metadata (which now has a separate filter file). I have now rebased this PR to include that change as well.